qiskit/transpiler/passes/synthesis/hls_plugins.py

```diff
@@ -1477,6 +1483,15 @@ class MCXSynthesisDefault(HighLevelSynthesisPlugin):
 
     This plugin name is :``mcx.default`` which can be used as the key on
     an :class:`~.HLSConfig` object to use this method with :class:`~.HighLevelSynthesis`.
+
+
+    The plugin supports the following plugin-specific options:
+
+    * ``optimization_metric``: The optimization metric, indicating whether we
+      should minimize the 2-qubit gate count or the T-count.
```

Comment:
Since the optimization metric could be anything, we should not document "2q or T count" here, but keep it a bit more generic
```suggestion
    * ``optimization_metric``: The optimization metric. This can e.g. indicate that the
      2-qubit gate count or the T-count should be minimized. See :class:`.OptimizationMetric`.
```
This occurs in some other parts below too 🙂

Reply:
Done in [1a91fee](https://github.com/Qiskit/qiskit/pull/14902/commits/1a91fee5efae8d8b02368215ab72aa2f1e105827).